include/MainWindow.h

```diff
@@ -168,6 +169,7 @@ public slots:
 	void toggleFxMixerWin();
 	void togglePianoRollWin();
 	void toggleControllerRack();
+    void updateCountDownDuration();
```

Comment:
Spaces instead of tabs for indentation. You want to change this right away so you don't have more work down the way :)